tests/lib/prepare.sh

```diff
@@ -260,8 +268,7 @@ prepare_memory_limit_override() {
         # systemd is backwards compatible so the limit is still set.
         cat <<EOF > /etc/systemd/system/snapd.service.d/memory-max.conf
 [Service]
-# mvo: disabled because of many failures in restore, e.g. in PR#11014
-#MemoryLimit=100M
+MemoryLimit=150M
```

Comment:
PR and description should mention memory limit is re-introduced

Reply:
done